demos/tests/run_tests.py

```diff
@@ -133,23 +148,32 @@ def option_to_args(key, value):
 
             print('Fixed arguments:', ' '.join(map(shlex.quote, fixed_args)))
             print()
+            device_args = demo.device_args(args.devices.split(' '))
```

Comment:
It would be better to use `split()` to allow an arbitrary number of spaces between devices.